src/app/header/header.component.scss

```diff
@@ -0,0 +1,29 @@
+ ::ng-deep .p-toolbar{
+  background-color: cadetblue;
```

Comment:
Also, remove small gap between sidenav and header.

![Uploading Screenshot 2021-05-24 at 12.58.41 PM.png…]()

Reply:
Removed space